rest/route/build_test.go

```diff
@@ -79,6 +97,9 @@ func (s *BuildByIdSuite) TestFindBuildById() {
 	s.Equal(utility.ToStringPtr("branch"), b.ProjectId)
 	s.Equal("task1", b.TaskCache[0].Id)
 	s.Equal(evergreen.TaskFailed, b.TaskCache[0].Status)
+	s.Equal("@daily", utility.FromStringPtr(b.DefinitionInfo.CronBatchTime))
+	s.NotNil(b.DefinitionInfo.BatchTime)
+	s.Equal(0, utility.FromIntPtr(b.DefinitionInfo.BatchTime))
```

Comment:
Can this test a batchtime number other than 0 since it's the default int value? It would also obviate the nil check right above this.

Reply:
I actually think testing 0 is stronger since it verifies that explicitly setting batchtime to 0 has different behavior than not setting it